internal/db/read_writer.go

```diff
@@ -525,6 +528,22 @@ func (rw *Db) SearchWhere(ctx context.Context, resources any, where string, args
 	return nil
 }
 
+// TransactionTimestamp returns the current transaction timestamp.
+func (rw *Db) TransactionTimestamp(ctx context.Context) (time.Time, error) {
```

Comment:
```suggestion
func (rw *Db) Now(ctx context.Context) (time.Time, error) {
```
I think `Now` seems more intuitive.